frontend-cert/js-projects/js-chat-bot/user-stories.md

```diff
@@ -0,0 +1,10 @@
+1. You should log a greeting message to the user, including a welcome statement from the bot introducing itself.
+1. You should create variables to store the bot's name, location, and favorite language.
+1. You should log a message to the user that includes the bot's name, location, and favorite language. For example: `Hi! I'm {botName}, I live in {botLocation}, and my favorite language is {favoriteLanguage}.`
+1. You should create a variable using `let` and assign it a fact about the bot's favorite language.
+1. You should log a message that combines a string and the fact stored in the variable to the user.
+1. You should reassign the variable you created earlier to a new fact about the bot's favorite language.
+1. You should log a message that combines a string and the new fact stored in the variable to the user.
+1. You should reassign the variable again to a final fact about the bot's favorite language.
+1. You should log a message using a template literal `` (`) `` to combine a string and the new fact stored in the variable.
```

Comment:
campers will not know about template literals at this point.
only string concatenation with the `+` operator